cfme/tests/cloud_infra_common/test_policy_simulation.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""This module tests events that are invoked by Cloud/Infra VMs."""
+import pytest
+
+from cfme.cloud.provider import CloudProvider
+from cfme.common.vm_views import PolicySimulationDetailsView
+from cfme.common.vm_views import PolicySimulationView
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+pytestmark = [
+    pytest.mark.usefixtures('uses_infra_providers', 'uses_cloud_providers', 'setup_provider'),
+    pytest.mark.tier(2),
+    pytest.mark.provider(
+        classes=[InfraProvider, CloudProvider],
+        required_fields=[(['cap_and_util', 'chargeback_vm'], 'cu-24x7')],
+        selector=ONE_PER_CATEGORY,
+        scope="module"
+    ),
+]
+
+
+@pytest.mark.parametrize("navigation", ["provider_vms", "vm_summary"])
+def test_policy_simulation_ui(appliance, provider, navigation):
+    """
+    Bugzillas:
+        * 1670456, 1686617, 1686619, 1688359, 1550503
+    Polarion:
+        assignee: jdupuy
+        casecomponent: Control
+        caseimportance: medium
+        initialEstimate: 1/6hr
+    """
+    collection = provider.appliance.provider_based_collection(provider)
+    vm = collection.instantiate('cu-24x7', provider)
+    if navigation == "provider_vms":
+        if BZ(1686617).blocks or BZ(1686619).blocks:
+            pytest.skip("Skipping due to blocker BZs, {}, {}".format(BZ(1686617), BZ(1686619)))
```

Comment:
Required:
You are using `BZ(1686617)` object... I think better to use simple string or `BZ(num).bug_id`

Reply:
More better if you put `uncollectif` with reason